add_column_to_s3_ds/README.md

```diff
@@ -1,3 +1,45 @@
-# Tinybird Versions - {{ YOUR USE CASE NAME HERE }}
+# Tinybird Versions - Add Column to an S3 Data Source
 
-Work in progress ...
+Adding a column to an S3 Data Source is a straightforward and commonly executed task in any Data Project.
+
+In this guide we'll show you how to do it fast and safe.
+
+> Remember to follow the [instructions](../README.md) to setup your Tinybird Data Project before jumping into the use-case steps
+
+## Step 1: Initialize a Nullable Column for 'environment'
```

Comment:
This is a picky thing, but if it is a one-step thing, we shouldn't say "Step 1: XXXX", let's explain the things the user needs to do, and that's it